examples/apps/cli/main.c

```diff
@@ -67,15 +71,21 @@ void otTaskletsSignalPending(otInstance *aInstance)
 }
 
 #if OPENTHREAD_POSIX && !defined(FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION)
-static void ProcessExit(void *aContext, uint8_t aArgsLength, char *aArgs[])
+static otError ProcessExit(void *aContext, uint8_t aArgsLength, char *aArgs[])
 {
     OT_UNUSED_VARIABLE(aContext);
     OT_UNUSED_VARIABLE(aArgsLength);
     OT_UNUSED_VARIABLE(aArgs);
 
     exit(EXIT_SUCCESS);
 }
-static const otCliCommand kCommands[] = {{"exit", ProcessExit}};
+
+static const otCliCommand kCommands[] = {
+    {"exit", ProcessExit},
+#if OPENTHREAD_EXAMPLES_SIMULATION
+    {"nodeidfilter", ProcessNodeIdFilter},
```

Comment:
What happens if we run `nodeidfilter` in virtual time simulation?

Reply:
`nodeidfilter` will not work in virtual time simulation. I will add the condition `OPENTHREAD_SIMULATION_VIRTUAL_TIME == 0` to control whether to add the user command `nodeidfilter`.